packages/server/graphql/public/types/AtlassianIntegration.ts

```diff
@@ -28,7 +34,18 @@ const AtlassianIntegration: AtlassianIntegrationResolvers = {
         projectKeyFiltersByCloudId[cloudId] = []
       })
     }
-    const issueRes = await manager.getIssues(queryString ?? null, isJQL, projectKeyFiltersByCloudId)
+    // Request one extra item to see if there are more results
+    const maxResults = first + 1
+    // Relay requires the cursor to be a string
+    const afterInt = parseInt(after ?? '-1', 10)
```

Comment:
+1 technically `after` is a string so could be anything, and then `parseInt` will return `NaN`. would be good to check agains that